apps/bfd-model/bfd-model-codebook-data/pom.xml

```diff
@@ -29,24 +33,29 @@
 			<artifactId>bfd-model-codegen-annotations</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+
+		<dependency>
+			<!-- Provides a sane facade for the giant mess of incompatible logging 
+				frameworks in Java. -->
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.glassfish.jaxb</groupId>
+			<artifactId>jaxb-runtime</artifactId>
+			<version>2.3.1</version>
```

Comment:
I think we need to pull this glassfish dependency version into dependencyManagement in a higher level pom since we use it in several places.